src/utils/general.js

```diff
@@ -34,6 +40,8 @@ const hasJsonStructure = str => {
 
 module.exports = {
   hasJsonStructure,
+  isError,
```

Comment:
Is this export done for something in one of the other PRs?

Reply:
Nope - good catch. Will remove

Reply:
> Looks good to me, do you know why the Node 10 build failed?

@conor-ward Yeah - should have known from the previous PR
It's because in Node10 you were not allowed to set arrow functions as methods on classes.

The problem with using es5 functions is that as we go deeper in scope (es5 functions create new scope) we lose reference to "this" keyword

So for that reason - all class methods must be declared using es5 syntax and we will also need to explicitly bind the methods to "this" if the method is used as a higher order function i.e(returns a function)
**Example:**
```
class Test {
  testFuncWithArrow() {
     return () => console.log(this)
  }
  testFuncWithES5() {
     return function() {
         console.log(this)
     }
  }
  useClassMethodInHigherOrderFunction() {
      return this.testFuncWithArrow
  }
}

var test = new Test()

test.testFuncWithArrow()() // logs the class
test.testFuncWithES5()() // logs undefined
test.useClassMethodInHigherOrderFunction()()() // logs undefined

classes are a messy business :D
```